plugins/CoreAdminHome/javascripts/optOut.js

```diff
@@ -1,22 +1,73 @@
 function submitForm(e, form) {
-    if (e.preventDefault) { // IE8 and below do not support preventDefault
```

Comment:
I haven't tested the feature yet but would have expected the preventDefault code and also below the setInterval is still needed?

Reply:
This code was to prevent the checkbox state from toggling until after the API call had been made, and then refresh the iframe to show the new checkbox state.  It seems a bit unnecessary when we are working with the first-party cookies, but if we do want to retain that workflow (i.e. sync the iframe form checkbox with the real world optout value after every change, to make sure that it reflects what will actually happen) I could put back the `preventDefault` and then fire a message back to the iframe after the first-party cookie has been set in the tracker JS.